packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -131,4 +137,62 @@ function logMetaData(changelog) {
   }
 }
 
+async function handleUnexpectedErrors(options, projectInfo, repository, e) {
+  const envInfo = await envinfo.run(
+    {
+      System: ['OS'],
+      Binaries: ['Node', 'Yarn', 'npm'],
+    },
```

Comment:
`Utilities > Git` can be interesting 
see values available: https://github.com/tabrindle/envinfo#cli-usage